test/e2e/tests/transaction/ens.spec.ts

```diff
@@ -15,200 +16,65 @@ describe('ENS', function (this: Suite) {
-    await mockServer
-      .forPost(infuraUrl)
-      .withJsonBodyIncluding({ method: 'eth_blockNumber' })
-      .thenCallback(() => ({
-        statusCode: 200,
-        json: {
-          jsonrpc: '2.0',
-          id: '1111111111111111',
-          result: '0x1',
-        },
-      }));
-    await mockServer
-      .forPost(infuraLineaMainnetUrl)
-      .withJsonBodyIncluding({ method: 'eth_blockNumber' })
-      .thenCallback(() => ({
-        statusCode: 200,
-        json: {
-          jsonrpc: '2.0',
-          id: '43',
-          result: '0x1',
-        },
-      }));
-    await mockServer
-      .forPost(infuraLineaSepoliaUrl)
-      .withJsonBodyIncluding({ method: 'eth_blockNumber' })
-      .thenCallback(() => ({
-        statusCode: 200,
-        json: {
-          jsonrpc: '2.0',
-          id: '43',
-          result: '0x1',
-        },
-      }));
-    await mockServer
-      .forPost(infuraSepoliaUrl)
-      .withJsonBodyIncluding({ method: 'eth_blockNumber' })
-      .thenCallback(() => ({
-        statusCode: 200,
-        json: {
-          jsonrpc: '2.0',
-          id: '43',
-          result: '0x29',
-        },
-      }));
-    await mockServer
-      .forPost(infuraUrl)
-      .withJsonBodyIncluding({ method: 'eth_getBalance' })
-      .thenCallback(() => ({
-        statusCode: 200,
-        json: {
-          jsonrpc: '2.0',
-          id: '1111111111111111',
-          result: '0x1',
-        },
-      }));
-    await mockServer
-      .forPost(infuraSepoliaUrl)
-      .withJsonBodyIncluding({ method: 'eth_getBalance' })
-      .thenCallback(() => ({
-        statusCode: 200,
-        json: {
-          jsonrpc: '2.0',
-          id: '6183194981233610',
-        },
-      }));
-    await mockServer
-      .forPost(infuraLineaMainnetUrl)
-      .withJsonBodyIncluding({ method: 'eth_getBalance' })
-      .thenCallback(() => ({
-        statusCode: 200,
-        json: {
-          jsonrpc: '2.0',
-          id: '6183194981233610',
-        },
-      }));
-    await mockServer
-      .forPost(infuraLineaSepoliaUrl)
-      .withJsonBodyIncluding({ method: 'eth_getBalance' })
-      .thenCallback(() => ({
-        statusCode: 200,
-        json: {
-          jsonrpc: '2.0',
-          id: '6183194981233610',
-        },
-      }));
-
-    await mockServer
-      .forPost(infuraUrl)
-      .withJsonBodyIncluding({ method: 'eth_getBlockByNumber' })
-      .thenCallback(() => ({
-        statusCode: 200,
-        json: {
-          jsonrpc: '2.0',
-          id: '1111111111111111',
-          result: {},
-        },
-      }));
-    await mockServer
-      .forPost(infuraLineaMainnetUrl)
-      .withJsonBodyIncluding({ method: 'eth_getBlockByNumber' })
-      .thenCallback(() => ({
-        statusCode: 200,
-        json: {
-          jsonrpc: '2.0',
-          id: '1111111111111111',
-          result: {},
-        },
-      }));
+    await mockMultiNetworkBalancePolling(mockServer);
```

Comment:
polling balance mocks are extracted in this new function, this way we'll be able to use it in other tests